tests/Feature/FlightControllerTest.php

```diff
@@ -2,547 +2,507 @@
 
-class FlightControllerTest extends TestCase
-{
-    use RefreshDatabase;
+uses(RefreshDatabase::class);
 
-    /** @test */
-    public function testGetFlightsForAnAirline()
-    {
-        City::factory(2)->create();
+it('gets flights for an airline ', function () {
+    $origin = City::factory()->create();
+    $destination = City::factory()->create();
 
-        Airline::factory()->create();
+    $airline = Airline::factory()->create();
 
-        Flight::create([
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
-            'departure_time' => '2023-05-05T14:30:00',
-            'arrival_time' => '2023-05-05T14:31:00',
-        ]);
-
-        $this->json('GET', action([FlightController::class, 'getFlights']))
-            ->assertSuccessful()
-            ->assertJsonFragment([
-                'airline_id' => 1,
-                'origin_city_id' => 1,
-                'destination_city_id' => 2,
-                'departure_time' => '2023-05-05T14:30:00',
-                'arrival_time' => '2023-05-05T14:31:00',
-            ]);
-    }
-
-    /** @test */
-    public function testCreateAFlightWithEmptyParameters()
-    {
-        City::factory(2)->create();
-        Airline::factory()->create();
-
-        $emptyAirlineData = [
-            'airline_id' => NULL,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
-            'departure_time' => '2023-09-11T11:18',
-            'arrival_time' => '2023-09-11T15:19'
-        ];
-        $this->json('POST', action([FlightController::class, 'store']), $emptyAirlineData)
-            ->assertStatus(422)
-            ->assertJsonValidationErrors('airline_id');
-
-        $emptyOriginData = [
-            'airline_id' => 1,
-            'origin_city_id' => NULL,
-            'destination_city_id' => 2,
-            'departure_time' => '2023-09-11T11:18',
-            'arrival_time' => '2023-09-11T15:19'
-        ];
-        $this->json('POST', action([FlightController::class, 'store']), $emptyOriginData)
-            ->assertStatus(422)
-            ->assertJsonValidationErrors('origin_city_id');
-
-        $emptyDestinationData = [
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => NULL,
-            'departure_time' => '2023-09-11T11:18',
-            'arrival_time' => '2023-09-11T15:19'
-        ];
-        $this->json('POST', action([FlightController::class, 'store']), $emptyDestinationData)
-            ->assertStatus(422)
-            ->assertJsonValidationErrors('destination_city_id');
+    Flight::factory()->create([
+        'airline_id' => $airline->id,
+        'origin_city_id' => $origin->id,
+        'destination_city_id' => $destination->id,
+    ]);
 
-        $emptyDepartureData = [
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
+    $this->get(route('flights.get'))
```

Comment:
add `Json` suffix, `getJson` instead of `get`.
That adds the `Accept: application/json` to the request, so we prevent unexpected behaviour